lib/format-context.js

```diff
@@ -2,18 +2,24 @@ const binding = require('../binding')
 const Stream = require('./stream.js')
 const ReferenceCounted = require('./reference-counted')
 const OutputFormat = require('./output-format')
+const IOContext = require('./io-context')
+const InputFormat = require('./input-format')
 
 class FFmpegFormatContext extends ReferenceCounted {
   constructor(io) {
     super()
 
-    this._io = io._ref()
+    if (io) {
+      this._io = io._ref()
+    }
```

Comment:
```suggestion
    this._io = io ? io._ref() : null
```

This way the object shape doesn't change.